src/modules/SystemSettings.rb

```diff
@@ -287,11 +289,17 @@ def device_supports_scheduler(device, scheduler)
     # @see Read
     def read_scheduler
```

Comment:
Let's take the opportunity to add `@return [Boolean]` :-)

Reply:
Sure!